### PR TITLE
Modify the documents for "vtk"

### PR DESCRIPTION
**Pull request by tianchiTJ**
Modify the documents for "vtk" based on the current code.

### CODE REVIEW

**Review by bodhinandach — APPROVED** [review]
Thanks @tianchiTJ

**Comment by jgiven100** [comment]
Wasn't this already taken care by @kks32? 
https://github.com/cb-geo/mpm-doc/commit/e862e7d34614db2c1f837c894271deb605a170d0
@bodhinandach @tianchiTJ

**Review by jgiven100 — APPROVED** [review]

File: user/postprocess/vtk.md [hunk 1]
```diff
@@ -5,13 +5,14 @@ When the CB-Geo mpm code is compiled with VTK libraries, the MPM code can be set
 ```JSON
   "post_processing": {
     "output_steps": 5,
+    "path": "results/",
+    "vtk" : ["stresses","velocities"],
```
Comment (kks32) [inline user/postprocess/vtk.md:9]: Could you add this line to https://mpm.cb-geo.com/#/user/preprocess/input as well?

**Review by kks32 — CHANGES_REQUESTED** [review]
Thanks Tianchi, could you please update the input part of the doc as well?

File: user/postprocess/vtk.md (hunk 1 above)
Comment (tianchiTJ) [reply user/postprocess/vtk.md:9]: Done!

**Review by kks32 — APPROVED** [review]

**Comment by kks32** [comment]
Thank you @tianchiTJ !

**Pull request merged**